### DISCUSSION TITLE
[预览] v2.0.10

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#2829] [分享 / Share]
- 修复专栏中视频卡片布局错位. (PR #2776 by [FoundTheWOUT](https://github.com/FoundTheWOUT))
- 尝试支持禁用 b 站切换播放器模式时的定位效果. (#483)
- 自定义顶栏:
  - 修复主站中的链接错误. (#2774)
  - 视频卡片的链接带上 `/video` 避免二次跳转. (#2779)

开发者相关:
- 稍后再看的错误处理统一移动到 core 中 (src/components/video/watchlater.ts)
- `isComponentEnabled` 能够对不存在的组件返回 `false` 了.
- 关闭 preview 分支的 CI 触发, 方便多个 PR 的合并.
- 对组件 / 插件的 watch task 默认启用 webpack 的 mode=development 以提供 source map.
- CONTRIBUTING 文档更新: 对于发起 PR 前的 commit, 仅提交源代码上的修改即可, 不建议把 dist 文件夹里的产物也提交, 否则容易在 PR 时产生冲突. (**现有的 PR 需要再合并下 preview 分支**)

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.0.10-preview'>[预览] v2.0.10</a>.</em>

**Comment by ChaXxl** [comment]
极简主页啥时候做好啊
![](https://i0.hdslb.com/bfs/article/15f9378faac85e74a155c5f236fd19e998ed5ca6.png@288w_288h_progressive.webp)

**Reply by the1812** [reply]
v1 的那种分区样式在宽屏下留白太多了我想改掉, ~~不过目前还没啥灵感~~ 现在有了, 在写了

**Reply by ChaXxl** [reply]
发个讨论让大家提意见啊，一起想

**Reply by xzk-kokodayo** [reply]
> 发个讨论让大家提意见啊，一起想

没毛病，咱们可能看不懂代码，但是UI还是可以帮忙设计的

**Reply by the1812** [reply]
有想法可以直接发啊, 讨论区里专门就有个 "分享" 类型.
